templates/s3-bucket.yaml

```diff
@@ -26,12 +26,6 @@ Parameters:
       - true
       - false
     Default: false
-  # only used with SameRegionResourceAccessToBucket, more info in AddSameRegionBucketDownloadRestriction.yaml
-  SameRegionResourceAccessToBucketGrantVpcAccess:
```

Comment:
since it's removed well need to update our PR examples that mention this parameter:

https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/4/files
https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/14/files

Reply:
I don't see either the now removed SameRegionResourceAccessToBucketGrantVpcAccess, or the remaining SameRegionResourceAccessToBucket parameter in those PR examples. Would you like me to create a PR to your forked iwanna-bucket branches to add the latter?

Reply:
sorry, you are correct.  I forgot we never added info about this since it's a rarely requested.